llvm/lib/Target/AMDGPU/BUFInstructions.td

```diff
@@ -119,6 +119,7 @@ class MTBUF_Real <MTBUF_Pseudo ps, string real_name = ps.Mnemonic> :
 
   let isPseudo = 0;
   let isCodeGenOnly = 0;
+  let hasPostISelHook = 1;
```

Comment:
Can this be removed now? Setting it on a Real seems weird anyway.

Reply:
I missed this one - it shouldn't be there. I'll remove it.